### DISCUSSION TITLE
Is it possible to make it so that instead of increasing the page size the text goes to a new line?

### DISCUSSION THREAD

**Discussion by deepestwaterdiver** [zim-desktop-wiki/zim-desktop-wiki#2652] [Q&A]
I'm not exactly sure the exact word for this, but is it possible to set up Zim so text doesn't just continue in the horizontal direction, but instead creates a new line?

**Comment by brucewestfall** [comment]
Are you referring to word wrap?  Mine does it already, not sure if it is from a plugin. Looking thru help files, only see a mention of it in source code plugin.
